### PR TITLE
Fix README instructions

### PR DESCRIPTION
**Pull request by nuzil**
Following current Installation Readme, I failed to install project.

1) bin/console replaced with app/console
2) demo mode now called "demo"

### CODE REVIEW

**Comment by fourty2** [comment]
seems like the demo is not up to date at all -- seems different to the clean install repository

File: README.md [hunk 1]
```diff
@@ -47,16 +47,16 @@ php -d memory_limit=-1 composer.phar create-project --no-dev ezsystems/ezplatfor
 Then, create a new database using the following command:
 
 ```bash
-php bin/console doctrine:database:create
+php app/console doctrine:database:create
```
Comment (andrerom) [inline README.md]: you seems to be installing 1.x and not 2.x or?

Comment (andrerom) [reply README.md]: v2 is using symfony 3.x, and hence need bin/console.

If you are looking for install instruction on 1.x, go to 1.13 branch. Or ideally online doc which should be the central place to go to.

**Comment by nuzil** [comment]
Hi all.
I just made git clone from master and the follow instructions.

**Comment by andrerom** [comment]
@nuzil looking at this repo, there is no console command in app here in master. Did you clone an old fork or otherwise checkout something from last year perhaps?

ref: https://github.com/ezsystems/ezplatform-demo/blob/master/bin/console

**Comment by nuzil** [comment]
Interesting,
Left part was create by:
- git clone
- composer install

Right Part:
- composer.phar create-project --no-dev ezsystems/ezplatform-demo

screen shot 2018-03-21 at 16 31 40

**Comment by damianz5** [comment]
@nuzil please try again with the `composer create-project [..]`
composer should show `Installing ezsystems/ezplatform-demo (v2.0.2)` now

**Comment by nuzil** [comment]
Thanks for support guys

**Comment by andrerom** [comment]
The tale of the missing tag ;) Sorry for the confusion @nuzil, hope it works better now :)

**Pull request closed without merging**